decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -42,7 +42,7 @@ def source_paths
                             desc: "Use a specific branch from GitHub's version"
 
       class_option :repository, type: :string,
-                                default: "https://github.com/decidim/decidim.git",
+                                default: "https://github.com/greenwoodt/decidim.git",
```

Comment:
Let's change back to decidim reference
```suggestion
                                default: "https://github.com/decidim/decidim.git",
```